api/samples/instrace_simple.c

```diff
@@ -234,7 +229,7 @@ event_app_instruction(void *drcontext, void *tag, instrlist_t *bb,
          * Using a fault to handle a full buffer should be more robust, and the
          * forthcoming buffer filling API (i#513) will provide that.
          */
-        IF_AARCHXX(&& !instr_is_exclusive_store(instr)))
+        IF_AARCHXX_ELSE(!instr_is_exclusive_store(instr), true))
```

Comment:
Oh, right.  I guess I'd put the whole if inside an ifdef (just not a fan of if (true)) but your call.